lua/telescope/_extensions/neorg/backlinks/file_backlinks.lua

```diff
@@ -0,0 +1,34 @@
+local neorg_loaded, neorg = pcall(require, "neorg.core")
+
+assert(neorg_loaded, "Neorg is not loaded - please make sure to load Neorg first")
+
+local file_backlink_regex = require("telescope._extensions.neorg.backlinks.utils")
+
+local function get_current_workspace()
```

Comment:
I feel like we got this function duplicated at quite a few places
perhaps extract it in the utils module here https://github.com/nvim-neorg/neorg-telescope/blob/main/lua/neorg/telescope_utils.lua